src/features/device-config/device-config.ts

```diff
@@ -150,5 +150,13 @@ export const generateConfig = async (
 		config.developmentMode = true;
 	}
 
+	const securebootInstaller = getBodyOrQueryParam(
+		req,
+		'secureboot',
+	)?.toString();
+	if (['true', 'on', '1'].includes(securebootInstaller || '')) {
+		config.installer = { secureboot: true };
```

Comment:
I've checked the fibery link but I don't see where it's documented that the format should be
```json
{ installer: { secureboot: true } }
```
Do you know where I would confirm that format is correct?

Reply:
It's here https://github.com/balena-os/meta-balena/blob/master/docs/uefi-secure-boot.md#provisioning